src/jit/compiler.h

```diff
@@ -252,8 +252,9 @@ class LclVarDsc
     unsigned char lvStackByref : 1;  // This is a compiler temporary of TYP_BYREF that is known to point into our local
                                      // stack frame.
 
-    unsigned char lvArgWrite : 1; // variable is a parameter and STARG was used on it
-    unsigned char lvIsTemp : 1;   // Short-lifetime compiler temp
+    unsigned char lvArgWrite : 1;         // there is at least one STLOC or STARG on this local
+    unsigned char lvMultipleArgWrite : 1; // there is more than one STLOC on this local
```

Comment:
There's an asymmetry between args and locals that I wrestled with -- args are implicitly written once, so a single `starg` or `ldarga` creates a multiple-def arg, whereas for locals we need to see two of `stloc` or `ldloca` to have a multiple def local. I could resolve this by setting `lvArgWrite` when the arg temp is allocated and update the logic for args and then the fields would have same meaning. Does that seem preferable?

`store` seems a bit too specific for the name since these also apply to address-of operators. I'd use `definition` but then the overlap with `lvISingleDef` would be even more confusing. How about `HasSingleWrite` and `HasMultipleWrites`?